frame/support/procedural/src/construct_runtime/expand/config.rs

```diff
@@ -78,8 +78,8 @@ pub fn expand_outer_config(
 			#fields
 		}
 
-		// GenesisConfig is in the process of being deprecated [https://github.com/paritytech/substrate/issues/14065]
 		#[cfg(any(feature = "std", test))]
+		#[deprecated(note = "Use `RuntimeGenesisConfig` instead.")]
```

Comment:
Should we add some info when it will be removed?

Reply:
We don't really know when, as it depends on releases but we can surely specify something like "removing it after xx/xx/xxxx". That would makes sense to me, wdyt?